package.json

```diff
@@ -1,93 +1,35 @@
 {
-  "name": "iframe-coordinator",
-  "version": "4.5.2",
-  "description": "Tools for coordinating embedded apps via iframes.",
-  "dependencies": {
-    "cheerio": "^1.0.0-rc.10",
-    "commander": "^2.20.3",
-    "decoders": "1.15.0",
-    "dev-cert-authority": "^1.1.1",
-    "express": "^4.17.3",
-    "find-root": "^1.1.0",
-    "http-proxy-middleware": "^1.3.1"
-  },
-  "files": [
-    "dist/client.js",
-    "dist/host.js",
-    "dist/index.js",
-    "dist/es5/client.js",
-    "dist/es5/host.js",
-    "dist/es5/index.js",
-    "dist/**/*.d.ts",
-    "cli/ifc-cli.js",
-    "cli/example-ifc.config.js",
-    "cli/embedded-app/dist/**/*"
+  "name": "iframe-coordinator-monorepo",
+  "version": "5.0.1-beta.1",
```

Comment:
Similar to the changelog. I think this should stay 4.5.2 until the CI build, so that standard-version is working from the last published version.